src/app/initialize/initialize.component.ts

```diff
@@ -0,0 +1,29 @@
+import { QuizService } from './../services/quiz.service';
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-initialize',
+  templateUrl: './initialize.component.html',
+  styleUrls: ['./initialize.component.css'],
+})
+
+export class InitializeComponent implements OnInit {
+  isValidFormSubmitted = false;
+
+  constructor(private quizService: QuizService, private route: Router) { }
+
+  ngOnInit(): void { }
+
+  // submit form results
+  onFormSubmit(form: NgForm) {
+    this.isValidFormSubmitted = true;
+    if (form.invalid) {
+      return;
+    }
+    this.quizService.createQuiz(form);
+    this.route.navigate(['/questions']);
```

Comment:
To abstract the navigation, you can create a navigation service and use something line 

```javascript
navigationService.goToQuestions();
```
This way, if the questions path changes in the future (e.g. from "/questions to /quiz/questions") you will only change this on one place.

Reply:
Done. Navigation service was implemented.